packages/store/src/internal/dispatcher.ts

```diff
@@ -32,7 +33,7 @@ export class InternalDispatcher {
   /**
    * Dispatches event(s).
    */
-  dispatch(actionOrActions: any | any[]): Observable<any> {
+  dispatch(actionOrActions: ActionType | ActionType[]): Observable<any> {
```

Comment:
Should it be a part of this PR? Since it's a public API.

Reply:
sure, I will be revert it